backend/tracim_backend/lib/search/elasticsearch_search/elasticsearch_search.py

```diff
@@ -429,6 +429,7 @@ def search_content(
             # important too, content of comment is less important. filename and file_extension is
             # only useful to allow matching "png" or "nameofmycontent.png".
             fields=es_search_fields,
+            quote_field_suffix=".keyword",
```

Comment:
Did you check a bit the result given by this ?
Not sure we should keep it now, until we do find a good way to use it, it has weird side effect for field that doesn't have sub-item named keywords: https://github.com/tracim/tracim/issues/2658#issuecomment-582807323

Reply:
I got this by merging the commit you told me to merge. I can remove it, I have no idea what it does.

Reply:
so, just remove it. ;-)

Reply:
Okay sir.

```suggestion
```